integration-tests/src/test/java/com/redhat/parodos/integration/notification/NotificationRecord.java

```diff
@@ -40,70 +42,170 @@ public void setUp() throws IOException {
+		});
+		assertEquals(401, e.getCode());
 	}
 
-	private void validateNotificationRecordCount(String testName, int notificationRecordsExpectedCount) {
-		logTestStep(testName, "Count Notification Records for the user");
-		assertDoesNotThrow(() -> {
-			Integer count = this.recordApiInstance.countUnreadNotifications("UNREAD");
-			log.info("Found ", count, "notification records for the user");
-			assertEquals(notificationRecordsExpectedCount, count.intValue());
+	@Test
+	public void notificationServiceGetAuthErr() {
+		String testName = "notificationServiceGetAuthErr";
+
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "");
+		apiClient.setBasePath(BASE_PATH);
+		this.recordApiInstance = new NotificationRecordApi(apiClient);
+
+		ApiException e = assertThrows(ApiException.class, () -> {
+			listNotificationRecord(testName, 0);
 		});
+		assertEquals(401, e.getCode());
 	}
 
-	private PageNotificationRecordResponseDTO validateNotificationRecordList(String testName,
-			int notificationRecordsExpectedCount) throws ApiException {
-		logTestStep(testName, "List Notification Records for the user");
-		PageNotificationRecordResponseDTO result = this.recordApiInstance.getNotifications(new Pageable(), null, null);
-		List<NotificationRecordResponseDTO> content = result.getContent();
-		log.info(content.toString());
-		assertEquals(notificationRecordsExpectedCount, content.size());
-		return result;
+	@Test
+	public void notificationServiceCountAuthErr() {
+		String testName = "notificationServiceCountAuthErr";
+
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "");
+		apiClient.setBasePath(BASE_PATH);
+		this.recordApiInstance = new NotificationRecordApi(apiClient);
+
+		ApiException e = assertThrows(ApiException.class, () -> {
+			countNotificationRecord(testName, 0);
+		});
+		assertEquals(401, e.getCode());
 	}
 
-	private void createNotificationMessage(String messageType, String body, String subject, String username) {
+	@Test
+	public void notificationServiceDeleteAuthErr() {
+		String testName = "notificationServiceDeleteAuthErr";
+
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "");
+		apiClient.setBasePath(BASE_PATH);
+		this.recordApiInstance = new NotificationRecordApi(apiClient);
+
+		ApiException e = assertThrows(ApiException.class, () -> {
+			deleteNotificationRecord(testName, UUID.randomUUID());
+		});
+		assertEquals(401, e.getCode());
+	}
+
+	@Test
+	public void notificationServiceUpdateAuthErr() {
+		String testName = "notificationServiceUpdateAuthErr";
+
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "");
+		apiClient.setBasePath(BASE_PATH);
+		this.recordApiInstance = new NotificationRecordApi(apiClient);
```

Comment:
These lines are repeated several times. WDYT to move them in a method annotated with `@BeforeAll`?

Reply:
The API is created in the @beforeAll, but only in the auth validation tests it recreated without the user

Reply:
The API is created in `@Before` not in `@BeforeAll` but I understand what you say.  That's correct.